src/Page/Form.purs

```diff
@@ -33,16 +40,26 @@ data Query a
   = Initialize a
   | EvalForm (Page Expr) a
   | UpdateKey Key ExprType a
+  | DropdownQuery DropdownSlot (Dropdown.Message Query ExprType) a
 
 type ParentInput = String
 
+type ChildQuery m = Coproduct1 (DropdownQuery m)
+
+type ChildSlot = Either1 DropdownSlot
+
+type DropdownQuery m = Dropdown.Query Query ExprType m
+
+type DropdownSlot = Key
```

Comment:
I don't feel strongly about this, but we could do away with the alias and just use `Key` directly for all our slots instead of aliasing it for every component we add
```purescript
type DateTimeSlot = Key
type DropdownSlot = Key
type SingleTypeaheadSlot = Key
type MultiTypeaheadSlot = Key
```
🤷‍♂️